prover/proof_fri_compressor/src/compressor.rs

```diff
@@ -202,4 +205,16 @@ impl JobProcessor for ProofCompressor {
             .await;
         Ok(())
     }
+
+    fn max_attempts(&self) -> u32 {
+        self.max_attempts
+    }
+
+    async fn get_job_attempts(&self, job_id: &L1BatchNumber) -> anyhow::Result<Option<u32>> {
+        let mut prover_storage = self.pool.access_storage().await.unwrap();
```

Comment:
Is `unwrap()` intentional here and in other implementations given that an error could be propagated?

Reply:
No, makes sense to propagate it